src/__test__/Calculator.test.js

```diff
@@ -0,0 +1,9 @@
+import renderer from 'react-test-renderer';
+import Calculator from '../components/Calculator';
+
+it('Calculator renders correctly', () => {
+  const tree = renderer
+    .create(<Calculator />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] This is the main component of the app that the user interacts with the most. It would therefore be nice if we added tests to simulate user interaction using the React Testing Library just like it is required of us in the requirements. The requirement for this kind of test is quoted below:

  > **_- Use React Testing Library to simulate user interaction_**
  
  ## Suggestion
  One user interaction we can implement is clicking a combination of keys and lastly the (=) key using the `fireEvent` method from `@testing-library/react` and then asserting that the result of that calculation is present on the page using the `getByText` method.
  
  This simple [GUIDE](https://www.pluralsight.com/guides/simulate-browser-events-react-react-testing-library) from Pluralsight can really help with this.